probe/base/base.go

```diff
@@ -123,12 +129,34 @@ func (d *DefaultOptions) Probe() probe.Result {
 	d.ProbeResult.PreStatus = d.ProbeResult.Status
 	d.ProbeResult.Status = status
 
+	d.ExportMetrics()
+
 	d.DownTimeCalculation(status)
 
 	d.ProbeResult.DoStat(d.Interval())
 	return *d.ProbeResult
 }
 
+// ExportMetrics export the metrics
+func (d *DefaultOptions) ExportMetrics() {
+	d.metrics.Total.With(prometheus.Labels{
+		"probe":  d.ProbeName,
+		"status": d.ProbeResult.Status.String(),
+	}).Inc()
+
+	d.metrics.Duration.With(prometheus.Labels{
+		"probe": d.ProbeName,
+	}).Set(d.ProbeResult.RoundTripTime.Seconds())
+
+	status := 1 // up
```

Comment:
minor beatification question but can we have something like `#define UP 1` (i dont know if an equivalent in go exists) so that we can do `status := UP // 1`  and `status = DOWN` ?